compiler/ballerina-treegen/src/main/java/io/ballerinalang/compiler/internal/treegen/TreeGen.java

```diff
@@ -57,11 +62,12 @@ public static void main(String[] args) {
         TreeGenConfig config = TreeGenConfig.getInstance();
         // 2) Get the syntax tree model by parsing the descriptor
         SyntaxTree syntaxTree = getSyntaxTree(config);
+        HashMap<String, TemplateNodeConfig> templateConfig = getTemplateConfig(config);
```

Comment:
Let's make this step 3) Get the syntax node metadata by parsing the metadata descriptor